From 0e591d6c1097f3d2f139f32c62d3ac5ed0761715 Mon Sep 17 00:00:00 2001
From: Duncan Macleod <macleoddm@cardiff.ac.uk>
Date: Fri, 4 Apr 2025 11:58:03 +0100
Subject: [PATCH 2/3] Handle no internet connection in test

---
 cwinpy/test/test_heterodyne.py | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/cwinpy/test/test_heterodyne.py b/cwinpy/test/test_heterodyne.py
index bd1b825..1452cab 100644
--- a/cwinpy/test/test_heterodyne.py
+++ b/cwinpy/test/test_heterodyne.py
@@ -729,9 +729,10 @@ transientTau = {tau}
             site="H1",
         )
 
-        assert int(data.t0.value) == 1126259460
-        assert data.dt.value == 1 / 4096
-        assert len(data) == 16384
+        if data is not None:
+            assert int(data.t0.value) == 1126259460
+            assert data.dt.value == 1 / 4096
+            assert len(data) == 16384
 
         del het
         del data
-- 
2.39.5

